app/templating/utils.py

```diff
@@ -0,0 +1,22 @@
+from app.questionnaire.rules import evaluate_when_rules
```

Comment:
Don't understand why the separate utils module.  It's very 'titles' centric so not really utils. Less cognitive load for the reader if it's just in `view_context.py` - unless there's a reason I'm missing (e.g. avoiding circ import).

Reply:
its called in several places, so we agreed in the utils would be best served

Reply:
Ideally this logic would form part of a `Question` view model of some type. But we don't have that concept and it's quite a big one to introduce.

Separate module is because it's used by both the context builder module and the summary view models.

And yep, there would have been a circular import if it was defined in `view_context.py` since that imports the summary view model which depends on this function.